app/server/server.ts

```diff
@@ -44,7 +44,17 @@ server.use(routes.core);
 server.use("/profile/", routes.profile);
 server.use("/api/", routes.api);
 server.use(routes.productsProvider("/api/"));
-server.use("/consent/", routes.consent);
+
+server.use(
+  "/consent/",
+  (req, res, next) => {
+    // we want this route to be loaded in an iframe
+    res.removeHeader("X-Frame-Options");
```

Comment:
I would highly recommend using [`Content-Security-Policy`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy) rather than `X-Frame-Options` and I would suggest limiting to Guardian domains.

Reply:
Yep, happy to look at `Content-Security-Policy` - I did begin looking at restricting iframe access to our domains. I believe the `X-Frame-Options` header is added by the `express` security library `helmetjs` that's being used here, I'm not sure this header plays alongside `Content-Security-Policy` so we may still need to remove it.

Reply:
@twrichards I've updated to set a `Content-Security-Policy` header.